src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -113,6 +107,6 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CppLibCreator>"$(_CppToolsDirectory)lib.exe"</CppLibCreator>
     </PropertyGroup>
 
-    <Error Condition="'$(_WhereCppTools)' != '0' AND '$(_VCVarsAllFound)' == '1'" Text="Platform linker not found. To fix this problem, download and install Visual Studio 2019 or Visual Studio 2022 from http://visualstudio.com. Make sure to install the Desktop Development for C++ workload. For ARM64 development also install C++ ARM64 build tools." />
+    <Error Condition="'$(_VCVarsAllFound)' == '1'" Text="Platform linker not found. To fix this problem, download and install Visual Studio 2019 or Visual Studio 2022 from http://visualstudio.com. Make sure to install the Desktop Development for C++ workload. For ARM64 development also install C++ ARM64 build tools." />
```

Comment:
Delete VS 2019 from this message? We do not VS2019 listed in https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/docs/prerequisites.md anymore.